index.js

```diff
@@ -1,33 +1,41 @@
 'use strict';
 
-var Dropbox = require('dropbox');
-var fs = require('fs');
+const Dropbox = require('dropbox'), fs = require('fs'), path = require('path');
 
-var token = '';
+function fileview(arg){
+	if(arg.upload) {upload(arg)}
```

Comment:
if there is one line statement in the `if` command, you can ignore the braces

```js
if(arg.upload) upload(arg)
```

or for better readability do

```js
if(arg.upload)
  upload(arg)
```